### PR TITLE
[systemtest] Refactor tracing STs and remove the abstract

### PR DESCRIPTION
**Pull request by im-konge**
### Type of change

- Refactoring

### Description

This PR refactors a bit our classes related to tracing. It:

1. removes `TracingAbstractST` as we currently have just one tracing type (if there will be another, we can bring it back easily)
2. adds few useful getters to `TestStorage`
3. creates `SetupJaeger` to clean up the code and have everything related to Jaeger in one place
4. removes few `if`s and methods that are not needed anymore, as we have just one tracing type

### Checklist

- [x] Make sure all tests pass

### CODE REVIEW

**Review by jankalinic — APPROVED** [review]

**Comment by im-konge** [comment]
@strimzi-ci run tests --cluster-type=ocp --cluster-version=4.13 --parallel-count=5--testcase=OpenTelemetryST --profile=regression

**Comment by im-konge** [comment]
@strimzi-ci run tests --cluster-type=ocp --cluster-version=4.13 --parallel-count=5 --testcase=OpenTelemetryST --profile=regression

**Review by jankalinic — APPROVED** [review]

File: systemtest/src/main/java/io/strimzi/systemtest/resources/jaeger/SetupJaeger.java [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+import static io.strimzi.systemtest.Constants.JAEGER_DEPLOYMENT_TIMEOUT;
+import static io.strimzi.systemtest.tracing.TracingConstants.CERT_MANAGER_CA_INJECTOR_DEPLOYMENT;
+import static io.strimzi.systemtest.tracing.TracingConstants.CERT_MANAGER_DEPLOYMENT;
+import static io.strimzi.systemtest.tracing.TracingConstants.CERT_MANAGER_NAMESPACE;
+import static io.strimzi.systemtest.tracing.TracingConstants.CERT_MANAGER_WEBHOOK_DEPLOYMENT;
+import static io.strimzi.systemtest.tracing.TracingConstants.JAEGER_INSTANCE_NAME;
+import static io.strimzi.systemtest.tracing.TracingConstants.JAEGER_OPERATOR_DEPLOYMENT_NAME;
+import static io.strimzi.test.k8s.KubeClusterResource.cmdKubeClient;
+
+public class SetupJaeger {
```
Comment (see-quick) [inline systemtest/src/main/java/io/strimzi/systemtest/resources/jaeger/SetupJaeger.java:36]: I think adding some JavaDoc would not hurt this class :)

**Review by see-quick — APPROVED** [review]
LGTM. Good job! 👍

**Review by Frawless — APPROVED** [review]

File: systemtest/src/main/java/io/strimzi/systemtest/resources/jaeger/SetupJaeger.java (hunk 1 above)
Comment (im-konge) [reply systemtest/src/main/java/io/strimzi/systemtest/resources/jaeger/SetupJaeger.java:40]: Done

**Pull request merged**